test/CosmoShips.t.sol

```diff
@@ -27,32 +28,47 @@ contract CosmoShipsTest is Test {
     }
 
     function testMintingFunctionality() public {
-        uint256 tokenId = 0;
+        IAttributeVerifier nftVerifier = new mockVerifier();
+        uint256 tokenId = 1;
+        CosmoShips testNft = new CosmoShips("0x1", tokenId, 1 ether, address(this), address(nftVerifier));
+
         uint256 attributes = 1096;
-        bytes32[2] memory t_proof = [
+        bytes32[4] memory t_proof = [
             bytes32(0xedc01e4d375758c57a25a6e5095e5ab59d5c2d87eb305682190981234d81175e),
-            0x1938d33112841f5ee65081f55d342198ab4a089d74cefcd0e7c616d43aad8c6d
+            0x1938d33112841f5ee65081f55d342198ab4a089d74cefcd0e7c616d43aad8c6d,
+            bytes32(0xedc01e4d375758c57a25a6e5095e5ab59d5c2d87eb305682190981234d81175b),
+            0x1938d33112841f5ee65081f55d342198ab4a089d74cefcd0e7c616d43aad8c6c
         ];
         bytes32[] memory proof = new bytes32[](2);
         proof[0] = t_proof[0];
         proof[1] = t_proof[1];
 
         vm.deal(user, 2 ether); // Provide ETH to user
         vm.prank(user);
-        nft.mint{value: 1 ether}(tokenId, attributes, proof);
+        testNft.mint{value: 1 ether}(attributes, proof);
```

Comment:
why testNft and not nft?

Reply:
Because it shadows the nft defined at the start of the file, and this gives a warning